flycheck.el

```diff
@@ -8711,6 +8715,25 @@ rustc command.  See URL `https://www.rust-lang.org'."
                (and (flycheck-buffer-saved-p)
                     (locate-dominating-file (buffer-file-name) "Cargo.toml"))))
 
+(flycheck-define-checker rust-cargo-clippy
+  "A Rust syntax checker and linter using clippy.
+
+This syntax checker needs the cargo clippy subcommand. See URL `https://github.com/Manishearth/rust-clippy'"
+  :command ("cargo" "clippy"
+            (eval flycheck-cargo-clippy-args)
+            "--"  "--error-format=json"
+            (option-list "-L" flycheck-rust-library-path concat)
+            (eval flycheck-rust-args))
```

Comment:
Can you explain the difference between `flycheck-cargo-clippy-args` and `flycheck-rust-args` with regards to this syntax checker?